.gitignore

```diff
@@ -4,3 +4,5 @@ typings
 *.tgz
 .idea
 .tmp
+dist
+app/build
```

Comment:
Shouldn't we build the application before demo? 
I don't see demo on github as a static code. Are you sure it should be in a repository? I don't know a context, that why I am asking :)